doc/source/install.rst

```diff
@@ -47,13 +47,43 @@ Execute the following command from the shell::
 
 Windows
 ```````
+Before installing the development version, uninstall the standard version of
```

Comment:
If someone uses a virtual env, no need to uninstall the version managed by anaconda or whatsoever. We can wait for other opinions, but I would just clarify this assumption.

Also, installing a dev version might require updates of some dependencies and/or installation of new dependencies. At least, a note along these lines would be good as well.